src/interfaces.ts

```diff
@@ -3,10 +3,11 @@ export type Files = Map<string, string>;
 export type FileTransform = (files: Files) => Promise<Files>;
 
 export enum VersionState {
-  ready = 'ready',
+  missing = 'missing',
   downloading = 'downloading',
-  unzipping = 'unzipping',
-  unknown = 'unknown',
+  downloaded = 'downloaded',
+  unzipping = 'installing',
+  ready = 'installed',
```

Comment:
I agree with this - the strings in the enum should match their string values

Reply:
Oh Cool! I felt it will be more intuitive for the user. (As `installing` is not much intuitive signifying that the version is actually being getting unzipped)